articles/node-vs-python/index.md

```diff
@@ -0,0 +1,179 @@
+<!--more-->
+
+### Table of Contents
+
+- [The History](#the-history)
+- [Node vs Python: The Differences](#node-vs-python:-the-differences)
+- [APIs in Node](#apis-in-node)
+- [APIs in Python](#apis-in-python)
+- [Further Reading](#further-reading)
+
+### The History
+
+#### Node.js
+Node.js is a server-side JavaScript runtime. JavaScript was originally designed to run within a browser, inside a JavaScript engine like V8. To expand the capabilities of JavaScript outside the browser, Node.js was developed in **2009** by **Ryan Dahl**. Node.js combined the capabilities of the V8 JavaScript engine, an event loop and low level calls into a robust server-side scripting runtime. It soon grew into the preferred language by web developers.
+
+#### Python
+Python is a multi-paradigm, scripting language conceived by **Guido van Rossum** in the late 1980s. Python is extremely easy to read, straightforward and simple to learn. It was built with simplicity in mind. It thus uses **indentation** to define scope of loops and functions. Python is extensively used as a server-side language. Frameworks like **Django** and **Flask** make it simple to design performant websites. It is also used to create APIs and Machine Learning models. Python, combined with the package manager **pip** make it perfect for projects.
+
+### Node.js vs Python: The Differences
+- [Performance and Architecture](#performance-and-architecture)
+- [Syntax and Ease of Use](#syntax-and-ease-of-use)
+- [Type of Projects](#type-of-projects)
+- [Frameworks and Expandability](#frameworks-and-expandability)
+- [Package Management](#package-management)
+
+#### Performance and Architecture
+Python and Node.js are interpreted languages. Therefore, they are comparatively slower than compiled languages like C++.
+
+Python is a single threaded and single flowing language. Though it is optimized by frameworks such as Django, it is slow. Python runs well on servers, but is quite unsuitable for mobile. The limit in processing power is a bottleneck to Python's performance.
+
+Node.js has the upper hand here. Node.js is heavily optimized by Google's V8 JavaScript Engine, making it performant. Node.js follows an event loop based architecture, which makes it non-blocking. Asynchronous requests can be handled. This means that the runtime can execute multiple requests without getting blocked.
+
+#### Syntax and Ease of Use
+Python has extremely readable syntax. This makes it the preferred programming language for beginners to work on. It omits the use of curly braces and uses indentation to define scope.
+
+Node.js follows the JavaScript syntax, which is harder to understand than Python. But for programmers that are familiar with C/C++, Node.js syntax is similar and poses no challenge.
+
+#### Type of Projects
+Both Python and Node.js are used for web development and full-stack projects.
```

Comment:
I would expound on this (web development vs full-stack projects) by and give examples.